lib/util/edit.ts

```diff
@@ -424,23 +475,19 @@ const processFile = ({
             break;
           }
           case 'whole': {
-            if (!item.file) {
+            if (!Export.WholeExportDeclaration.isFileFound(item)) {
```

Comment:
I had to create a type guard because `TypeScript` couldn't determine that `item.file` couldn't be `undefined` in my `deeplyGetExportNames` function